.bazelci/presubmit.yml

```diff
@@ -25,7 +25,19 @@ platforms:
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_sh_toolchain_test"
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_no_cc_config_run_in_container_test"
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_no_cc_config_sh_toolchain_test"
-    # docker_autoconfig tests
+    test_flags:
+    - "--action_env=RBE_AUTOCONF_ROOT=/workdir"
+    - "--action_env=PATH"
+    - "--test_output=errors"
+    - "--verbose_failures"
+    - "--extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/latest:toolchain_docker"
+    - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
+    - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
+    - "--platforms=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
+  docker_autoconfig:
```

Comment:
Not sure why you decided to break off the docker_autoconfig to run on a non rbe platform? unless you have a reason, keep them together with the same platform as above.

Reply:
No particular reason. I basically wanted this PR to demonstrate that the new format, using task instead of platform, can support multiple of these tasks.
I will abandon this PR and instead go ahead with https://github.com/bazelbuild/bazel-toolchains/pull/671 which adds a separate new task .